:memo: Clarify how to enable Blackfire depending on plan (#2869)
platformsh/platformsh-docs: docs/src/increase-observability/integrate-observability/blackfire.md

### On a Grid plan or {{% names/dedicated-gen-3 %}} infrastructure 

> If you have subscribed to a Grid plan complete with the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
> the [Blackfire automated integration](#automated-integration) is enabled on your environments by default. 

Reviewer: :hammer_and_wrench: I am not sure about the `complete with` language. And maybe `Grid plan` isn't great because Grid is a kind of environment, not really a plan. It'd be nice if we had more clarity on terminology there :disappointed: 

```suggestion
If you have Grid environments on a plan with the Platform.sh [Observability Suite](https://platform.sh/features/observability-suite/),
the [Blackfire automated integration](#automated-integration) is enabled on your environments by default. 
```

> If you have subscribed to a Grid plan without the Observability suite,
> or if you are a {{% names/dedicated-gen-3 %}} user,
> the Blackfire automated integration isn't enabled on your environments.
> To enable Blackfire on your PHP or Python apps,

Reviewer: :hammer_and_wrench: To shorten things a little and focus on what to do, rather than what doesn't happen.

```suggestion
If you have Grid environments without the Observability Suite or {{% names/dedicated-gen-3 %}} environments,
you need to enable the integration yourself.
To enable Blackfire on your PHP or Python apps,
```
